services/galley/src/Galley/API/Internal.hs

```diff
@@ -438,29 +440,50 @@ sitemap = do
 
 rmUser :: UserId -> Maybe ConnId -> Galley ()
 rmUser user conn = do
-  let n = unsafeRange 100 :: Range 1 100 Int32
-  tids <- Data.teamIdsForPagination user Nothing (rcast n)
+  let n = toRange (Proxy @100) :: Range 1 100 Int32
+      nRange1000 = rcast n :: Range 1 1000 Int32
+  tids <- Data.teamIdsForPagination user Nothing n
```

Comment:
Don't you need to loop through the pages here too?

Reply:
I left the team deletion code the same (modulo replacing `rcast n` with `n`, where `n` is slightly differently defined, but should have the same value.

Perhaps it would be worth paging through teams as well. My hunch is that whoever wrote this code, assumed a user wouldn't be in more than 100 teams, hence there shouldn't be the next page to process.